rfhub2/cli/statistics_importer.py

```diff
@@ -45,28 +54,23 @@ def get_execution_files_paths(self) -> Set[Path]:
             if self._is_valid_execution_file(p)
         }
 
-    def add_statistics(self, statistics: List[KeywordStatistics]) -> Tuple[int, int]:
+    def add_statistics(
+        self, statistics: List[KeywordStatistics], execution_file: Path
+    ) -> int:
         """
         Adds statistics from provided list to app.
         :param statistics: List of KeywordStatistics objects
-        :return: list of dictionaries with collection name and number of keywords.
+        :param execution_file: Path to file from statistics where extracted
+        :return: number of statistics sent to app
         """
-        collections, keywords = set(), set()
-        for stat in statistics:
-            stat_req = self.client.add_statistics(KeywordStatisticsList.one(stat))
-            if stat_req[0] == 201:
-                collections.add(stat.collection)
-                keywords.add(".".join((stat.collection, stat.keyword)))
-            elif stat_req[0] != 400:
-                print(stat_req[1]["detail"])
-                raise StopIteration
-            else:
-                print(
-                    f"""Record already exists for provided collection: {
-                    stat.collection}, keyword: {stat.keyword} and execution_time: {stat.execution_time}"""
-                )
-
-        return len(collections), len(keywords)
+        stats_req = self.client.add_statistics(KeywordStatisticsList.of(statistics))
+        if stats_req[0] == 201:
+            return len(statistics)
+        elif stats_req[0] != 400:
+            print(stats_req[1]["detail"])
+        else:
+            print(f"""Records already exist for file from {execution_file}""")
```

Comment:
You could add unit test for this branch, then the PR would not decrease coverage. You can also remove `KeywordStatisticsList.one` method because it is substituted everywhere with `KeywordStatisticsList.of` method now.

Reply:
Done. Coverage results are impacted mostly by cli.py, which length has increased. This one is covered fully by accptance tests, so maybe. we could exclude it wih proper annotation?

Reply:
Thanks, I know, but still wanted to improve it where it is easy to do. I would prefer to add the unit tests for cli in the future instead.

Reply:
I'm all for it. Last time I struggled with mocking cli, but maybe there is some proper way to do it, that I do not know of.